src/docfx/Errors.cs

```diff
@@ -4,12 +4,14 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace Microsoft.Docs.Build
 {
     internal static class Errors
     {
+#pragma warning disable SA1124 // DoNotUseRegions
+
+        #region Level Error
```

Comment:
will this impose additional maintenance burden? if an info is turned into warning we'll have to do the house keep work to move it to a difference location?